x-pack/plugins/alerting/server/saved_objects/mappings.ts

```diff
@@ -124,33 +126,34 @@ export const alertMappings: SavedObjectsTypeMappingDefinition = {
     mutedInstanceIds: {
       type: 'keyword',
     },
-    meta: {
-      properties: {
-        versionApiKeyLastmodified: {
-          type: 'keyword',
-        },
-      },
-    },
+    // NO NEED TO BE INDEXED
+    // meta: {
+    //   properties: {
+    //     versionApiKeyLastmodified: {
+    //       type: 'keyword',
+    //     },
+    //   },
+    // },
     monitoring: {
       properties: {
         run: {
           properties: {
```

Comment:
@XavierM Sorry I'm a bit late with this comment since the PR has been already merged, but:

Until it's too late and we can't introduce non-additive changes to the mappings anymore, can we consolidate and refactor the `monitoring` and `lastRun` objects? These two objects contain similar data, and it's confusing for a lot of devs, including me, @jpdjere, @maximpn etc.

You can see how our code looks like that has to deal with these two objects, it's more complex than should be:

https://github.com/elastic/kibana/blob/7e633bb063051f60e5c26a1ed9fa243c2cd5b1bb/x-pack/plugins/security_solution/server/lib/detection_engine/rule_monitoring/logic/rule_execution_log/create_rule_execution_summary.ts